sgf-viewer/src/content.ts

```diff
@@ -61,6 +61,99 @@ export const pages: object =
                 "victim": "Latest (cp505-v1-MCTS), 64 visits",
                 "description": ["With 64 visits, KataGo's <code>Latest</code> network plays at the level of a  <a href=\"https://arxiv.org/pdf/2211.00241.pdf#page=17\">top-20 human professional</a>. We achieve a win rate of 48% against this victim simply by increasing the number of adversary visits to 8192. The adversary wins by the same qualitative strategy of staking out a corner, but plays significantly more stones in the victim's territory. The adversary loses when the victim plays the game out to the end, resulting in a very full board."]
+            },
+            {
+                "title": "KataGo without search with pass-alive defense",
+                "dir_name": "no_search_hardened",
+                "server": "dqn.ist.berkeley.edu",
+                "_path_comment": "Computed by `grep -n cp505h-v1 * | shuf | head -n 6 | cut -f1,2 -d:` in /nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs",
+                "paths_with_line_num": [
+                    {
+                        "path": "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs/BEB88CB5DC883E28.sgfs",
+                        "line": 3
+                    },
+                    {
+                        "path": "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs/699C18B496BE1116.sgfs",
+                        "line": 6
+                    },
+                    {
+                        "path": "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs/BDEA7D5310AB3A4A.sgfs",
+                        "line": 7
+                    },
+                    {   "path": "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs/88F51AE755AD0B72.sgfs",
+                        "line": 5
+                    },
+                    {   "path": "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs/F23AA96ACE3D6889.sgfs",
+                        "line": 8
+                    },
+                    {   "path": "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-20221108/43-extra/sgfs/B510F069013F3D36.sgfs",
+                        "line": 2
+                    }
+                ],
+                "max_games": 10,
+                "adversary": "349 million training steps, 600 visits",
+                "victim": "Latest (cp505-v1-MCTS), no search, pass-alive defense",
+                "description": ["We hardcode a defense for KataGo by making the victim not be able to pass until it has no more legal moves outside its territory. With more training, however, we are able to find another attack against the victim, achieving a win rate of 99%. The adversary gets the victim to form a <a href=\"https://senseis.xmp.net/?Dragon\">dragon</a> and then kills it."]
```

Comment:
probably should write a better description here eventually, i didn't think hard about this